packages/rancher-vsphere/rancher-vsphere-csi/generated-changes/patch/templates/controller/deployment.yaml.patch

```diff
@@ -0,0 +1,20 @@
+--- charts-original/templates/controller/deployment.yaml
++++ charts/templates/controller/deployment.yaml
+@@ -22,7 +22,7 @@
+     spec:
+       serviceAccountName: vsphere-csi-controller
+       {{- if .Values.csiController.nodeSelector }}
+-      nodeSelector:
++      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
```

Comment:
if nodeSelectors are not supplied, this code worked by using affinity.

If nodeSelectors are supplied, this code was broken on Windows clusters since default node-selectors are not applied for Windows.